source/ui/PreferencesMenu.hx

```diff
@@ -79,7 +81,9 @@ class PreferencesMenu extends ui.OptionsState.Page
 
         if (getPref("fps-counter"))
             FlxG.stage.removeChild(Main.fpsCounter);
-
+        if (getPref("fullscreen")){
```

Comment:
oops. I was trying to make a fullscreen toggle and it didn't work. I accidentally left it here